Formula/imagemagick.rb

```diff
@@ -72,49 +73,42 @@ def install
       --enable-static
     ]
 
-    if build.without? "modules"
-      args << "--without-modules"
-    else
-      args << "--with-modules"
-    end
-
-    if build.with? "opencl"
-      args << "--enable-opencl"
-    else
-      args << "--disable-opencl"
-    end
-
-    if build.with? "openmp"
-      args << "--enable-openmp"
-    else
-      args << "--disable-openmp"
-    end
-
-    if build.with? "webp"
-      args << "--with-webp=yes"
-    else
-      args << "--without-webp"
+    %W[
+      hdri hdri
+      opencl opencl
+      openmp openmp
+      zero-configuration zero-configuration
+    ].each_slice(2) do |option, feature|
+      args << (build.with?(option) ? "--enable-#{feature}" : "--disable-#{feature}")
     end
 
-    if build.with? "openjpeg"
-      args << "--with-openjp2"
-    else
-      args << "--without-openjp2"
+    %W[
+      fftw fftw
+      fontconfig fontconfig
+      freetype freetype
+      ghostscript gslib
+      jpeg jpeg
+      liblqr lqr
+      libpng png
+      librsvg rsvg
+      libtiff tiff
+      libwmf wmf
+      little-cms lcms
+      magick-plus-plus magick-plus-plus
+      modules modules
+      openexr openexr
+      openjpeg openjp2
+      pango pango
+      perl perl
+      threads threads
+      webp webp
+      x11 x
+    ].each_slice(2) do |option, feature|
+      args << (build.with?(option) ? "--with-#{feature}" : "--without-#{feature}")
     end
 
-    args << "--without-gslib" if build.without? "ghostscript"
-    args << "--with-perl" << "--with-perl-options='PREFIX=#{prefix}'" if build.with? "perl"
-    args << "--with-gs-font-dir=#{HOMEBREW_PREFIX}/share/ghostscript/fonts" if build.without? "ghostscript"
-    args << "--without-magick-plus-plus" if build.without? "magick-plus-plus"
-    args << "--enable-hdri=yes" if build.with? "hdri"
-    args << "--without-fftw" if build.without? "fftw"
-    args << "--without-pango" if build.without? "pango"
-    args << "--without-threads" if build.without? "threads"
-    args << "--with-rsvg" if build.with? "librsvg"
-    args << "--without-x" if build.without? "x11"
-    args << "--with-fontconfig=yes" if build.with? "fontconfig"
-    args << "--with-freetype=yes" if build.with? "freetype"
-    args << "--enable-zero-configuration" if build.with? "zero-configuration"
+    args << "--with-gs-font-dir=#{HOMEBREW_PREFIX}/share/ghostscript/fonts" if build.with? "ghostscript"
```

Comment:
you can put the `HOMEBREW_PREFIX/"share/ghostscript/fonts"` in a useless variable. We do that all the time for this reason :D